frontend/plan/components/modals/CalendarModal.tsx

```diff
@@ -1,137 +1,156 @@
+            <Row>
+                <Col>
+                    <ICSButton>ICS URL</ICSButton>
                 </Col>
-                <Col span={8}>
+                <Col>
                     <FixedInput
                         type="text"
                         readOnly
                         value={url}
                         onClick={(e) => (e.target as HTMLInputElement).select()}
                     />
                 </Col>
-                <Col className="col control">
-                    <a
-                        className="button is-info"
+                <Col>
+                    <CopyButton
                         onClick={async () => {
                             try {
                                 await navigator.clipboard.writeText(url);
-                                toast.info("Copied to clipboard!");
+                                showToastMessage(
+                                    "Successfully copied to clipboard!"
+                                );
                             } catch (error) {
-                                toast.error(
+                                showToastMessage(
                                     "Failed to copy! You need to manually copy the URL."
                                 );
                             }
                         }}
                     >
                         Copy
-                    </a>
+                    </CopyButton>
                 </Col>
             </Row>
-
-            <hr />
-
-            <Row className="columns has-text-centered">
-                <div className="column">
-                    <h3><b>Import to Google Calendar</b></h3>
-                    <BigText>
-                        Use the URL above to import to Google Calendar. Need
-                        help?
-                        <br />
-                        <a
-                            href="https://support.google.com/calendar/answer/37100#add_via_link"
-                            target="_blank"
-                            rel="noreferrer noopener"
-                        >
-                            Check out this guide!
-                        </a>
-                    </BigText>
-                </div>
-                <div className="column">
-                    <h3><b>Import to macOS Calendar</b></h3>
-                    <BigText>
-                        Use the URL above to import to the macOS Calendar app.
-                        Need help?
-                        <br />
-                        <a
-                            href="https://support.apple.com/guide/calendar/subscribe-to-calendars-icl1022/mac"
-                            target="_blank"
-                            rel="noreferrer noopener"
-                        >
-                            Check out this guide!
-                        </a>
-                    </BigText>
-                </div>
-            </Row>

```

Comment:
Is there a reason why we're removing this? Seems like it would be helpful to include for people that don't know how to use ICS URL